ci-operator/config/openshift/elasticsearch-operator/openshift-elasticsearch-operator-master.yaml

```diff
@@ -125,6 +129,8 @@ tests:
         name: pipeline:logging-elasticsearch6
       - env: IMAGE_LOGGING_KIBANA6
         name: pipeline:logging-kibana6
+      - env: IMAGE_LOG_FILE_METRIC_EXPORTER
```

Comment:
Not needed here as this is the e2e test for the EO that does not invoke the CLO logic

Reply:
fixed

Reply:
fixed, thx